src/OSPSuite.FuncParserNative/src/FuncNode.cpp

```diff
@@ -94,9 +94,6 @@ namespace FuncParserNative
    {
       assert(_nodeType != NT_INVALID);
 
-      if (_firstOperand != NULL)
-         delete _firstOperand;
-
```

Comment:
that was not a good idea and resulted in crashes in some simulations.
Because the simplifying procedure rearranges nodes
https://github.com/Open-Systems-Pharmacology/OSPSuite.FuncParser/blob/9f6edec4500982baa5a559e25c4701609d9b8c6a/src/OSPSuite.FuncParserNative/src/FuncNode.cpp#L440-L451
(and they may not be deleted of course)

Reply:
I don't understand.